features/step_definitions/site_assertion_steps.rb

```diff
@@ -98,3 +98,8 @@
 Then(/^I should be redirected to the organisation dashboard$/) do
   i_should_be_on_the_path organisation_path(@site.organisation)
 end
+
+Then(/^the last version should list "([^"]*)" as the author of the change$/) do |name|
+  expect(@site.versions.last.whodunnit).to eql(name)
```

Comment:
I thought about doing this the other day, checking records in feature specs. I had a Google and it seem like most people would expect these to only exercise things the user can see.

Reply:
I think you changed my mind on this. I think it's worth ignore the "ideal" standard if the test is still useful, which it was in this case!

Especially with the fact that this is not in the UI as you said

Reply:
Oh, just when I'd come over to your side! ;) The request spec reproduced the bug too, though, so that's all right.

Reply:
Hm, maybe I'll amend the commit to go into detail about that, so future devs beware of controller specs.